src/main/java/net/snowflake/client/jdbc/SnowflakePreparedStatement.java

```diff
@@ -28,6 +30,20 @@ public interface SnowflakePreparedStatement {
    */
   void setBigInteger(int parameterIndex, BigInteger x) throws SQLException;
 
+  /**
+   * Sets the designated parameter to the given <code>java.sql.Array</code> object. The driver
+   * converts this to an SQL <code>ARRAY</code> value when it sends it to the database.
+   *
+   * @param parameterIndex the first parameter is 1, the second is 2, ...
+   * @param x an <code>Array</code> object that maps an SQL <code>ARRAY</code> value
+   * @param x an <code>SnowflakeType</code> object that maps an Snowflake type
+   * @exception SQLException if parameterIndex does not correspond to a parameter marker in the SQL
+   *     statement; if a database access error occurs or this method is called on a closed <code>
+   *     PreparedStatement</code>
+   * @throws SQLFeatureNotSupportedException if the JDBC driver does not support this method
+   * @since 1.2
```

Comment:
is it copied from the JDBC docs? we should validate that since the info here may be misleading